core-api/src/main/java/com/optimizely/ab/config/audience/UserAttribute.java

```diff
@@ -67,7 +67,8 @@ public Object getValue() {
         return value;
     }
 
-    public @Nullable Boolean evaluate(ProjectConfig config, Map<String, ?> attributes) {
+    public @Nullable
+    Boolean evaluate(ProjectConfig config, Map<String, ?> attributes) {
```

Comment:
Same as other comments about annotation placement for return types. This is formatting is worse.